pkg/cmd/container/exec.go

```diff
@@ -181,3 +184,11 @@ func generateExecProcessSpec(ctx context.Context, client *containerd.Client, con
 
 	return pspec, nil
 }
+
+func setInitialTTYSize(pspec *specs.Process) {
```

Comment:
please [see](https://github.com/containerd/nerdctl/pull/2121/files#r1146919388)

Reply:
I think it's a function works more like specs.Process' decorator,  so it's not so proper to go to consoleutil